compiler/plugins/target/AMD-AIE/iree-amd-aie/Transforms/AMDAIEAssignTiles.cpp

```diff
@@ -365,6 +367,19 @@ LogicalResult assignNonLocalTiles(RewriterBase &rewriter, Operation *op,
             llvm::map_to_vector(logicalObjectFifo.getTiles(), [](Value tile) {
               return dyn_cast_if_present<TileOp>(tile.getDefiningOp());
             });
+
+        auto fromMemrefOp = dyn_cast<AMDAIE::LogicalObjectFifoFromMemrefOp>(
```

Comment:
Could you add a comment here explaining that you're limiting the number of tile options to buffer count to avoid repeated accesses of the same buffer being assigned to different tiles?